src/libraries/Microsoft.PowerFx.Core/Texl/IExpressionRewriter.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx
+{
+    /// <summary>
+    /// Expression re-writer interface. This will be leveraged by InitalFixup method to rewrite the expression in LSP.
+    /// </summary>
+    public interface IExpressionRewriter
+    {
+        public string Process(CheckResult check);
```

Comment:
>string [](http://example.com/codeflow?start=15&length=6)

can you comment this further - 
Assuming this returns an expression for rewritten result?